Cargo.toml

```diff
@@ -15,29 +15,29 @@ path = "src/main.rs"
 lto = true
 
 [dependencies]
+base64 = "^0.10.0"
+byteorder = "1.1"
 clap = "2.26"
-users = "0.7"
-slog-term = "2.2"
-slog-async = "2.1"
-slog-scope = "4.0"
+error-chain = { version = "0.12", default-features = false }
+hostname = "0.1"
+hyper = "0.11"
+ipnetwork = "0.12"
 mime = "0.3"
-serde = "1.0"
-serde_derive = "1.0"
-serde-xml-rs = "0.2"
-serde_json = "1.0"
+nix = "0.9"
+openssh-keys = "0.3"
 openssl = "0.9.17"
-base64 = "0.9"
-byteorder = "1.1"
 pnet = "0.21"
 reqwest = "0.7"
-hyper = "0.11"
-error-chain = { version = "0.12", default-features = false }
-openssh-keys = "0.3"
-update-ssh-keys = "0.3.0"
-ipnetwork = "0.12"
-hostname = "0.1"
+serde = "1.0"
+serde-xml-rs = "0.2"
+serde_derive = "1.0"
+serde_json = "1.0"
+slog-async = "2.1"
+slog-scope = "4.0"
+slog-term = "2.2"
 tempdir = "0.3"
-nix = "0.9"
+update-ssh-keys = "0.3.0"
+users = "0.7"
```

Comment:
Minor: looks like this commit also alphabetizes the dependency list too? I usually split those out as a separate commit and explicitly mention there are no functional changes. Otherwise, mixing lots of movements with other changes makes it harder to tell what really changed.

Reply:
Indeed. I've split that into a dedicated earlier commit.